frontend/lib/src/components/widgets/DataFrame/DataFrame.tsx

```diff
@@ -703,18 +873,27 @@ function DataFrame({
                 sticky: false,
                 tint: true,
               },
-              rowMarkerTheme: {
-                bgCell: theme.bgHeader,
-                bgCellMedium: theme.bgHeader,
+              rowMarkers: {
+                kind: "checkbox",
+                checkboxStyle: "square",
+                theme: {
+                  bgCell: theme.bgHeader,
+                  bgCellMedium: theme.bgHeader,
+                },
```

Comment:
is this a duplicate of [these lines](https://github.com/streamlit/streamlit/pull/8411/files#diff-eda1f0faafab9bbafe2c7b8b34241c35192488443ab3e02f0291d91e40ef632fR835-R842)? Is there a way to move it to a variable like

```typescript
const rowMarkers = (isRowSelectionActivated || element.editingMode === DYNAMIC) && {
    rowMarkers: {
      kind: "checkbox",
      checkboxStyle: "square",
      theme: {
        bgCell: theme.bgHeader,
        bgCellMedium: theme.bgHeader,
    },
}
```

Reply:
Hmm, I think it is a bit of a coincidence that it's the same, but this might not be the case in the future. In a previous state of this PR, we already had differences, like using the rounded checkbox style in certain situations.